api/index.js

```diff
@@ -53,9 +53,61 @@ async function fetchStats(username) {
 
   const user = res.data.data.user;
 
+  // FETCH ALL COMMITS
+  let commitPromises = user.contributionsCollection.contributionYears.map(
+    async (year) => {
+      // Don't fetch contributions older than 2008 (Perf optimization)
```

Comment:
ah thanks 💯 😆

Reply:
Yup! fixed it real quick. since github was first started on 2008 i wouldn't care about older commits.